src/connectors.rs

```diff
@@ -1159,6 +1159,8 @@ pub fn builtin_connector_types() -> Vec<Box<dyn ConnectorBuilder + 'static>> {
         Box::new(impls::crononome::Builder::default()),
         Box::new(impls::s3::Builder::default()),
         Box::new(impls::kafka::consumer::Builder::default()),
+        #[cfg(feature = "grpc")]
+        Box::new(impls::grpc::client::Builder::default())
```

Comment:
Does this mean, we can only have 1 connector that runs any number of grpc services provided in the `PROTO_FILES` env var per compiled tremor binary? If we want to have, say, two different connectors from two different sets of proto files, would this be possible? 

Can you clarify what exactly can be run with tremor given the setup here? How is the grpc connector scoped?

What can we do to have different connector types per service? Or somehow group services on the command line? One connector type per proto file?

Reply:
Right now, we have one connector that talks to all the services defined in all the provided files under `PROTO_FILES`. I don't know whether it'd be possible to have one connector per file, but it'd definitely require a lot of refactoring and restructuring.